docs/modeling-your-data/running-data-models-via-snowplow-bdp/dbt/using-dbt/index.md

```diff
@@ -160,8 +160,20 @@ In the first step, you can provide the data model name, a description, and the o
 
 ![](images/data-model-create-step-1.png)
 
+When selecting a `dbt-core` version, below you'll find a table which contains the exact `dbt-adapter` versions that are bundled with each specific version of `dbt-core`:
 
-Please note that the model name needs to match to the corresponding datamodeling subfolder name in GitHub. The dbt project for the model in the above example would therefore live under:
+| dbt-core 	| BigQuery 	| Databricks 	| Postgres 	| Redshift 	| Snowflake 	|
```

Comment:
How will we keep this up to date?

Reply:
I'm not sure but I think it's important that it's out there somewhere as it isn't documented anywhere 😅 I think dbt's release velocity will slow down, and so we'll be bumping versions later and later in Console so this shouldn't be something that's updating super frequently.

I also think if this is public people will expect it to be up to date and might ping on Discourse or wherever else to remind us if things aren't updated, so that could be a feedback loop to ensure we're updated as well!

Reply:
@jbeemster is there a defined process for releasing new DBT versions into console? Could updating this table be part of it if so

Reply:
Not that I am aware of... ultimately SRE gets told we need a new version and then we add it into the backend when requested.  Who does the requesting and updating of docs I am not entirely sure!